azure/terraform.sh

```diff
@@ -76,7 +95,17 @@ function other_actions() {
 }
 
 function state_output_taint_actions() {
-  terraform $action $other
+  if [ "$action" == "tflist" ]; then
+    download_tool "tflist"
```

Comment:
can we download the tool only if not already installed?

Reply:
done.
The following if
```
if [ ! -n "$(command -v tflist)" ] && [ ! -f "tflist" ]; then
```
checks if the tool is installed on the computer or if it is present in the current folder